spec.html

```diff
@@ -31407,9 +31408,8 @@ <h1>BigInt.prototype.toString ( [ _radix_ ] )</h1>
           1. Let _x_ be ? thisBigIntValue(*this* value).
           1. If _radix_ is *undefined*, let _radixMV_ be 10.
           1. Else, let _radixMV_ be ? ToIntegerOrInfinity(_radix_).
-          1. If _radixMV_ &lt; 2 or _radixMV_ &gt; 36, throw a *RangeError* exception.
-          1. If _radixMV_ = 10, return ! ToString(_x_).
-          1. Return the String representation of this Number value using the radix specified by _radixMV_. Letters `a`-`z` are used for digits with values 10 through 35. The precise algorithm is implementation-defined, however the algorithm should be a generalization of that specified in <emu-xref href="#sec-numeric-types-bigint-tostring"></emu-xref>.
+          1. If _radixMV_ is not in the inclusive interval from 2 to 36, throw a *RangeError* exception.
+          1. Return BigInt::toString(_x_, _radixMV_).
```

Comment:
Should this be `!`’d? It seemingly can’t throw and the other `BigInt::toString()` does have an explicit `!`.

Reply:
No, ever since we changed how completion records are used, this is unnecessary for AOs that don't return completion records. The other call site's use of `!` should be removed though, so thanks for the catch.

Reply:
By the way, ecmarkup normally would catch these cases for us, but I'm guessing it isn't tracking these AOs because of their strange names. @bakkot?

Reply:
It's not the name. Rather, linting rules like that one only apply within algorithm steps, which this isn't, as far as ecmarkup is concerned.

Reply:
Shouldn't algorithm steps inside methods also be considered algorithm steps?

Reply:
Sorry, I'm referring specifically to the [table under ToString](https://tc39.es/ecma262/multipage/abstract-operations.html#sec-tostring), which does not have anything to identify that it contains algorithm steps.

Reply:
ah right, good point